core/domain/role_services.py

```diff
@@ -108,6 +129,17 @@
 }
 
 
+def get_human_readable_role(role):
+    """Converts role to human readable format.
+
+    Args:
+        role: str. The role to convert.
+    Returns:
+        str. Role in human readable format.
+    """
+    return role.lower().replace('_', ' ')
```

Comment:
In that comment, I had pulled out VIEWABLE and UPDATABLE roles and mentioned them explicitly.
Should we have an explicit dict for human readable roles as well?

That will add one more change for admin while adding, removing role.